src/trials.jl

```diff
@@ -262,12 +273,40 @@ function prettymemory(b)
     return string(@sprintf("%.2f", value), " ", units)
 end
 
-Base.summary(io::IO, t::Trial) = print(io, "Trial(", prettytime(time(t)), ")")
-Base.summary(io::IO, t::TrialEstimate) = print(io, "TrialEstimate(", prettytime(time(t)), ")")
-Base.summary(io::IO, t::TrialRatio) = print(io, "TrialRatio(", prettypercent(time(t)), ")")
-Base.summary(io::IO, t::TrialJudgement) = print(io, "TrialJudgement(", prettydiff(time(ratio(t))), " => ", time(t), ")")
+function withtypename(f, io, t)
+    needtype = get(io, :typeinfo, Nothing) !== typeof(t)
+    if needtype
+        print(io, nameof(typeof(t)), '(')
+    end
+    f()
+    if needtype
+        print(io, ')')
+    end
+end
+
+_summary(io, t, args...) = withtypename(() -> print(io, args...), io, t)
+
+Base.summary(io::IO, t::Trial) = _summary(io, t, prettytime(time(t)))
+Base.summary(io::IO, t::TrialEstimate) = _summary(io, t, prettytime(time(t)))
+Base.summary(io::IO, t::TrialRatio) = _summary(io, t, prettypercent(time(t)))
+Base.summary(io::IO, t::TrialJudgement) = withtypename(io, t) do
+    print(io, prettydiff(time(ratio(t))), " => ")
+    printtimejudge(io, t)
+end
+
+_show(io, t) =
+    if !(get(io, :verbose, false)) && get(io, :compact, true)
```

Comment:
It looks like `:verbose` is added in #121 by simply copying the old `verbose` keyword argument.  Now that the standard `:compact` IO context is supported (as of #121), it seems redundant and I suggest to remove it.  What do you think?

Reply:
Sure!

Reply:
Good. It's removed now.